cmd/consumer/responses.go

```diff
@@ -239,6 +239,11 @@ func injectMessageResponses(ma *handler.MessageActions) {
 		`- <https://golang.org/conduct>`,
 		`If you have any questions or concerns please reach out in <#C4U9J9QBT> or email support@gobridge.org.`,
 	)
+
+	ma.HandleStatic("ask", "how to ask questions", []string{"don't ask", "dont ask", "dontask", "just ask", "justask"},
+		`Don't ask to ask. Just ask. We'll tell you let you know if there's a better place to ask.`,
```

Comment:
It looks like you restructured this sentence and left a piece behind. I think you meant to remove `tell you`.

Reply:
yep